src/components/molecules/ZopaFooter/ZopaFooter.tsx

```diff
@@ -394,6 +394,8 @@ export interface FooterProps extends HTMLAttributes<HTMLDivElement> {
   mainCustomLegalCopy?: string | string[];
   additionalCopy?: string[];
   customFooterLinks?: CustomFooterLink[];
+  isCobranded?: boolean;
+  MainCobrandedLegalCopy?: any; //TODO: add type
```

Comment:
struggling to type this

Reply:
casing inconsistency

```suggestion
  mainCobrandedLegalCopy?: FooterProps['customFooterLinks']
```

I think is the type you want here

Reply:
`Type '() => React.JSX.Element' is not assignable to type 'CustomFooterLink[]'.`